Brewfile.lock.json

```diff
@@ -2,40 +2,40 @@
   "entries": {
     "brew": {
       "clang-format": {
-        "version": "19.1.1",
+        "version": "19.1.2",
```

Comment:
`m`: I would prefer these changes here in an extra PR, please.

Reply:
Reverted ✅